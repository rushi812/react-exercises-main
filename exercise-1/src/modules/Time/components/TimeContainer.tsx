import Time from "./Time"

const TimeContainer: React.FC = () => {
  return <Time />
}

export default TimeContainer
