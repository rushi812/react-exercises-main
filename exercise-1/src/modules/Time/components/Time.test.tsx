import TimeContainer from './TimeContainer'
import renderer, { act } from 'react-test-renderer';

describe('App Rendering', () => {

  it('Should match the snapshot', () => {
    jest.useFakeTimers()
    const appComponent = renderer.create(<TimeContainer />)
    act(() => {
      jest.advanceTimersByTime(10000 * 60);
    })

    expect(appComponent).toMatchSnapshot()
  })
})