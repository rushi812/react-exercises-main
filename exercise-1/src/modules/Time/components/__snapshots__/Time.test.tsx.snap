// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Rendering Should match the snapshot 1`] = `
<div
  className="timeContainer"
>
  11:10:29 pm
</div>
`;
